.travis.yml

```diff
@@ -29,11 +29,11 @@ after_success:
     npm run coverage:upload;
   fi
   if [[ $TRAVIS_BRANCH == 'release' && $TRAVIS_PULL_REQUEST == 'false' ]]; then
```

Comment:
hmm. I don't like having any logic in travis.yml scripts now when we have repo-cooker.
I'd like to see here only repo-cooker script invocation. Th rest of the logic shoul go into actions.